identity-service/src/notifications/index.js

```diff
@@ -122,39 +143,53 @@ class NotificationProcessor {
         })
       }
       // Delay 3s
-      await new Promise(resolve => setTimeout(resolve, 3000))
+      await new Promise(resolve => setTimeout(resolve, NOTIFICATION_INTERVAL_SEC))
 
-      done()
+      done(error)
```

Comment:
apart from being correct, passing error to done doesn't impact error handling in any way right?

Reply:
nope! it's the first arg here and it defaults to `null` https://github.com/OptimalBits/bull#basic-usage if you don't provide it